subprojects/entitled-mirror/containers/sync/Containerfile.sync

```diff
@@ -0,0 +1,9 @@
+FROM registry.access.redhat.com/ubi8/nginx-118:latest
+
+USER root
+
+RUN dnf install yum-utils -y
+
+COPY sync_commands.sh /scripts/
+RUN ["chmod", "+x", "/scripts/sync_commands.sh"]
+ENTRYPOINT ["/scripts/sync_commands.sh"]
```

Comment:
could you move this file out of the container, so that the list of repo to synchronize can be provided as a ConfigMap (I guess that's the easiest)

I was going to suggest "pass the list of repo to enable in arguments and iterate over it", but
```
reposync -p /repo --download-metadata --repoid rhel-8-for-x86_64-baseos-rpms
reposync -p /repo --download-metadata --repoid rhel-8-for-x86_64-baseos-eus-rpms --releasever=8.2
```
this second one makes that harder!

and I don't know if failing (`set -e`) is the best option or not, my pod is looping forever on the script, and it takes a while to get back to the actual line of the problem